.changeset/honest-apples-guess.md

```diff
@@ -0,0 +1,21 @@
+---
+"@frontity/analytics": minor
+"@frontity/components": minor
+"@frontity/comscore-analytics": minor
+"@frontity/connect": minor
+"@frontity/core": minor
+"frontity": minor
+"@frontity/google-ad-manager": minor
+"@frontity/google-analytics": minor
+"@frontity/google-tag-manager-analytics": minor
+"@frontity/head-tags": minor
+"@frontity/hooks": minor
+"@frontity/html2react": minor
+"@frontity/mars-theme": minor
+"@frontity/smart-adserver": minor
+"@frontity/twentytwenty-theme": minor
+"@frontity/types": minor
+"@frontity/yoast": minor
```

Comment:
@nicholasio was thinking if these updates require a minor update or rather a patch instead? These are not changing public facing API but rather the environment the code is running on. What do you think?

Reply:
That's a good point, I think I didn't go with a patch because this is not exactly a bug fix, but I'm fine to change it to patch